### PR TITLE
session 45

### PR DESCRIPTION
**Pull request by pkuligowski**

### CODE REVIEW

File: sessions/45/tasklist.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    [tc.ListFiles(21, '/'), Send, WaitMode.Wait],
+
+    # Telemetry download between session 41 and 44
+    [tc.DownloadFile(22, '/telemetry.current', [i for i in range(1300, 1500, 25)]), Send, WaitMode.Wait],
+
+    # RadFET Experiment data download
+    [tc.DownloadFile(23, '/radfet', [i for i in range(0, 8, 1)]), Send, WaitMode.Wait],
+    [tc.DownloadFile(24, '/radfet', [i for i in range(8, 16, 1)]), Send, WaitMode.Wait],
+
+    # Goodbye satellite with 1200
```
Comment (begreeen) [inline sessions/45/tasklist.py]: just wrong comment xD

Comment (pkuligowski) [reply sessions/45/tasklist.py]: why :p

Comment (pkuligowski) [reply sessions/45/tasklist.py]: aaaaa

Comment (pkuligowski) [reply sessions/45/tasklist.py]: true

**Pull request merged**